### DISCUSSION TITLE
Does cypress support chained not?

### DISCUSSION THREAD

**Discussion by quad5** [cypress-io/cypress#9598] [Questions and Help] [answered]
Code:
> cy.get('pref-ings')
.find(.bubble-group .account-bubble)
.not(.hide)
.not(.type-more)

It works in browser's console as:
`document.querySelectorAll('.prefs-ings .bubble-group .account-bubble:not(.hide):not(.type-more)')`

![image](https://user-images.githubusercontent.com/2043287/101678022-381a4480-3a12-11eb-9160-c90c182718be.png)
![image](https://user-images.githubusercontent.com/2043287/101678049-423c4300-3a12-11eb-98aa-1c957d290585.png)
<img width="857" alt="image" src="https://user-images.githubusercontent.com/2043287/101678141-6566f280-3a12-11eb-8a9f-2f320f4a8f63.png">

**Comment by jennifer-shehane** [comment, marked as answer]
Yes, it does support chaining multiple nots. 

`index.html`

```html
<html>
  <body>
    <div class="foo">Foo
      <div class="bar type-more">Foo 1</div>
      <div class="bar hide">Foo 2</div>
      <div class="bar">Foo 3</div>
      <div class="bar type-more">Foo 4</div>
      <div class="bar hide">Foo 5</div>
    </div>
    <div class="baz">Baz
      <div class="bar">Foo 1</div>
      <div class="bar">Foo 2</div>
      <div class="bar">Foo 3</div>
      <div class="bar">Foo 4</div>
      <div class="bar">Foo 5</div>
    </div>
  </body>
</html>
```

`spec.js`

```js
it('test', () => {
  cy.visit('index.html');
  cy.get('.foo')
    .find('.bar')
    .not('.hide')
    .not('.type-more')
    .should('contain', 'Foo 3')
})
```

<img width="673" alt="Screen Shot 2020-12-11 at 5 30 42 PM" src="https://user-images.githubusercontent.com/1271364/101895947-9db72e00-3bd6-11eb-9fb1-18eb4ec5e8e6.png">
